stable/memcached/templates/statefulset.yaml

```diff
@@ -50,14 +50,21 @@ spec:
         imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
         command:
         - memcached
-        - -m {{ .Values.memcached.maxItemMemory  }}
+        - -m {{ .Values.memcached.maxItemMemory }}
         {{- if .Values.memcached.extendedOptions }}
         - -o
         - {{ .Values.memcached.extendedOptions }}
         {{- end }}
         {{- if .Values.memcached.verbosity }}
         - -{{ .Values.memcached.verbosity }}
         {{- end }}
+        {{- range $key, $value := .Values.memcached.extraArgs }}
```

Comment:
Can you change this to use `with`? See an example on line 20.

Reply:
Using `with` statement now. The `extraArgs` value had to be changed from `map` to `list`.